docs/Community/sig-sgx/OWNERS

```diff
@@ -0,0 +1,17 @@
+chair:
+  - TBD
+
+reviewers:
```

Comment:
I think our original list of committers and areas of ownership are outdated. This is not specific to SGX. We should review those areas and then perhaps figure out the list of approvers in various areas. That can be done after this doc is merged and is not a blocker, but just something the CGC needs to do.